### PR TITLE
Finished menu section and responsive design for tablet and desktop

### PR DESCRIPTION
**Pull request by Constantin-Matei**
…ktop.

### CODE REVIEW

File: src/_component/connect/connect.scss [hunk 1]
```diff
@@ -83,7 +90,7 @@
   [class^="icon__"] {
     width: 100%;
     height: 100%;
-    font-size: 10vw;
+    font-size: 8vw;
```
Comment (QuintyHH) [inline src/_component/connect/connect.scss:93]: I wouldn't use viewport for font-size directly, read up this article on Fluid Typography
https://www.smashingmagazine.com/2016/05/fluid-typography/

**Review by QuintyHH — APPROVED** [review]

**Review by vasilemaftita — APPROVED** [review]
Nice; Work...

**Review by biggicode — APPROVED** [review]

**Review by RalucaRbc — APPROVED** [review]

**Comment by simonamarcu2** [comment]
💯

**Review by simonamarcu2 — COMMENTED** [review]
Nice work 💯

**Review by robertoalx — APPROVED** [review]
Yeyyyy more commits, you are legend mate :) 💯

**Pull request merged**